### PR TITLE
Initial implementation of the configuration

### PR DESCRIPTION
**Pull request by tailhook**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -95,13 +97,19 @@ fn _main() -> anyhow::Result<()> {
...
+    let cfg = config::get_config();
 
     let mut builder = env_logger::Builder::from_env(
         env_logger::Env::default().default_filter_or("warn")
     );
     log_levels::init(&mut builder, &opt);
     builder.init();
 
+    let cfg = cfg.unwrap_or_else(|e| {
+        log::warn!("Config error: {:#}", e);
```
Comment (1st1) [inline src/main.rs:109]: Can you rephrase it to `Error in config file '/path/to/config/cli.toml': error text here`?

**Review by 1st1 — APPROVED** [review]
lgtm. Can we also print (in grey) something like "applied /path/to/edgedb/cli.toml configuration file" when a user starts the repl?

**Comment by 1st1** [comment]
Also let's rename `limit` to `implicit_limit` as @elprans suggested, but that can be done in a separate PR if that's more convenient.

File: src/main.rs (hunk 1 above)
Comment (tailhook) [reply src/main.rs:109]: Config file path is added here: https://github.com/edgedb/edgedb-cli/pull/481/files/9a3f395c2c0958204d24f064676302fe24e3a4ea#diff-cba64c21ab992eaad29fce147a08f4560a4769bc14682b8a96081a5fd02dbecdR46
(`#[context(..)]` decorator around `read_config`)

**Comment by tailhook** [comment]
> Also let's rename `limit` to `implicit_limit` as @elprans suggested, but that can be done in a separate PR if that's more convenient.

Then we should rename `\set limit` ->  `\set implicit-limit`, right?
